.github/workflows/ci.yml

```diff
@@ -74,8 +74,20 @@ jobs:
         working-directory: __tests__/e2e
         env:
           TERM: xterm # this is needed for tput
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         with:
-          name: test-results
+          name: test-results-e2e-test
           path: "**/junit.xml"
           if-no-files-found: error
+
+  merge-test-results:
```

Comment:
What's the reason for this? Plan to replace https://github.com/elastic/synthetics/blob/main/.github/workflows/test-reporter.yml?

Reply:
Thanks for reminding me about this.

I only learned this is not necessary in the next PRs.

Reply:
https://github.com/elastic/synthetics/pull/950/commits/07289f7d6a9a7eb40f7e2c1a2eddcb030563f767